lib/uniara_virtual_parser/models/absence.rb

```diff
@@ -0,0 +1,12 @@
+module UniaraVirtualParser
```

Comment:
Fix the indentation, please. We use 2 spaces for tabs. 
Check this https://github.com/bbatsov/ruby-style-guide